### PR TITLE
Jetbrains installation routine

### PR DESCRIPTION
**Pull request by javis86**
Hi people! 
I've added the routine for jetbrains installation via dotnet tools. I  fixed command titles too.
Regards.
Javier

### CODE REVIEW

**Comment by javis86** [comment]
Solves 2 issues:
[Rename [fakesharper] to FakeSharper:](https://github.com/fakesharper/fakesharper/issues/12)
[Install ReSharper command line tool via .NET Core](https://github.com/fakesharper/fakesharper/issues/13)

**Comment by jaqra** [comment]
@javis86 thanks for nice work. i will review and merge soon

File: package.json [hunk 1]
```diff
@@ -39,31 +39,31 @@
 		"commands": [
 			{
 				"command": "fakesharper.inspectcode",
-				"title": "[fakesharper] Inspect Code"
+				"title": "FakeSharper: Inspect Code"
```
Comment (jaqra) [inline package.json]:
```suggestion
				"title": "Inspect Code",
				"category": "FakeSharper"
```

**Review by ghost — COMMENTED** [review]
May this better?

File: package.json (hunk 1 above)
Comment (javis86) [reply package.json]: Ok. I will read the documentation about this topic!

**Review by ghost — DISMISSED** [review]
Thank you so much. Lastly can you please remove `package-lock.json`. Sorry for one by one review

**Comment by javis86** [comment]
I'm wondering if the fact of keeping package-lock.json tracked. I am not familiarized with npm package but i had the same question about it.
Here the doc of package-lock.json. 
[package-lock.json](https://docs.npmjs.com/cli/v6/configuring-npm/package-lock-json)

> This file is intended to be committed into source repositories, and serves various purposes:
> 
> Describe a single representation of a dependency tree such that teammates, deployments, and continuous integration are guaranteed to install exactly the same dependencies.
> Provide a facility for users to "time-travel" to previous states of node_modules without having to commit the directory itself.
> To facilitate greater visibility of tree changes through readable source control diffs.
> And optimize the installation process by allowing npm to skip repeated metadata resolutions for previously-installed packages.
>

**Comment by jaqra** [comment]
i mean we already have `yarn.lock` file

**Review by ghost — APPROVED** [review]
thank you!

**Pull request merged**